bitblas/ops/operator.py

```diff
@@ -50,26 +51,36 @@ def __init__(self, config: OperatorConfig):
         assert self.is_valid_config(config), (f"Invalid config for {self.__class__.__name__}: "
                                               f"{config}")
         self.config = config
+        self.kernel_name = ""
```

Comment:
The name of this value `kernel_name` is a bit ugly.

Should `BaseKernelNameGenerator` class keeps a `kernel_name` attribute?

Reply:
Thanks, this attribute has been replaced with a variable in "generate_valid" function